src/ir_York.cpp

```diff
@@ -0,0 +1,358 @@
+// Copyright 2022 Daniele Gobbetti
+/// @file
+/// @brief Support for the York AC protocol (AC model: MHH07P17 remote GRYLH2A).
+
+// Supports:
+//   Brand: York,  Model: MHH07P17
+// Note: Most of the code is autogenerated by the provided tools or assembled
+// from other support classes
+
+#include "ir_York.h"
+#include <algorithm>
+#include <cstring>
+#ifndef ARDUINO
+#include <string>
+#endif
+#include "IRrecv.h"
+#include "IRremoteESP8266.h"
+#include "IRsend.h"
+#ifdef UNIT_TEST
+#include "IRsend_test.h"
+#endif
+#include "IRtext.h"
+#include "IRutils.h"
+
+
+
+
+using irutils::addBoolToString;
+using irutils::addModeToString;
+using irutils::addFanToString;
+using irutils::addTempToString;
+using irutils::addSwingVToString;
+using irutils::addLabeledString;
+using irutils::minsToString;
+
+
+// Constants
+const uint16_t kHdrMark = 4887;
+const uint16_t kBitMark = 612;
+const uint16_t kHdrSpace = 2267;
+const uint16_t kOneSpace = 1778;
+const uint16_t kZeroSpace = 579;
+const uint16_t kFreq = 38000;  // Hz. (Guessing the most common frequency.)
+const uint16_t kOverhead = 3;
+
+#if SEND_YORK
+// Alternative >64bit function to send YORK messages
+// Function should be safe over 64 bits.
+/// Send a  formatted message.
+/// Status: ALPHA / Untested. The Checksum logic is missing
+/// @param[in] data An array of bytes containing the IR command.
+///                 It is assumed to be in MSB order for this code.
+/// e.g.
+/// @code
+///   uint8_t data[kStateLength] = {0x10, 0x08, 0xE0, 0x40, 0x02, 0x10, 0x80, 0x10, 0x80, 0x12, 0x00, 0x00, 0x00, 0x00, 0x37, 0xA0, 0x37};
+/// @endcode
```

Comment:
You can remove this, as
a) It's a note for the programmer generated from the automatic generator.
b) You've determined it is in LSB order.